### PR TITLE
 migrated library module in v11

### PR DESCRIPTION
**Pull request by Arpit-SerpentCS**

### CODE REVIEW

File: library/models/library.py [hunk 1]
```diff
@@ -676,7 +676,6 @@ class LibraryBookRequest(models.Model):
     _name = "library.book.request"
     _rec_name = 'req_id'
 
-    @api.multi
     @api.depends('type')
```
Comment (AnuKPatel) [inline library/models/library.py:675]: @Arpit-SerpentCS  Add For loop in compute field's method.

File: library/models/product.py [hunk 2]
```diff
@@ -263,8 +215,10 @@ def _compute_books_availablity(self):
     lang = fields.Many2one('product.lang', 'Language')
     editor_ids = fields.One2many('book.editor', "book_id", "Editor")
     author = fields.Many2one('library.author', 'Author')
-    code = fields.Char(compute_="_product_code", method=True,
-                       string='Acronym', store=True)
+    code = fields.Char(compute_="_product_code",
```
Comment (AnuKPatel) [inline library/models/product.py:217]: @Arpit-SerpentCS Manage line of code. Arrange the code in minimum lines.

**Comment by JayVora-SerpentCS** [comment]
not fixed yet

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Remove product module in the manifest file. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/__manifest__.py#L12

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Improve the README.rst file of a module. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/README.rst

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS What is the use of this domain? 'categ_id' field is not defined in the class https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/wizard/book_issue_no_view.xml#L13

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Add domain in inside the search condition. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/wizard/card_no.py#L17

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Add active_id inside the browse method. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/wizard/terminate_reason.py#L16

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Improve this line. `book_rec.name = rec.name.id`  https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/wizard/update_book.py#L24

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS  No need of data if noupdate="0". https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/security/library_security.xml#L4

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please use DEFAULT_SERVER_DATETIME_FORMAT instead of `"%Y-%m-%d %H:%M:%S"` in everywhere in the code.  https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_library/library/models/library.py#L223

**Comment by Arpit-SerpentCS** [comment]
@murtuzasaleh Please review the code. Made improvements and tested according to comments.

cc @JayVora-SerpentCS

**Pull request merged**